### PR TITLE
Update FileSystemInfo.xml

### PR DESCRIPTION
**Pull request by sabarish98**
## Summary

Changed summary for FileSystemInfo.LinkTarget to provide better clarity on the property.

Fixes dotnet/runtime#62166

### CODE REVIEW

File: xml/System.IO/FileSystemInfo.xml
Comment (gewarren) [inline xml/System.IO/FileSystemInfo.xml:1322]:
```suggestion
        <value>The target path of the symbolic link located in <see cref="P:System.IO.FileSystemInfo.FullName" />, or <see langword="null" /> if this <see cref="T:System.IO.FileSystemInfo" /> instance doesn't represent a link.</value>
```

**Comment by sabarish98** [comment]
> If the instance represents a link, then wouldn't a link have to exist at FullName? I think these are one and the same?

@gewarren : Yes, that's correct. But when we work with file paths dynamically, I feel we will use FullName to find the full path since we don't know with what value the instance was created with. So, I feel, it's valid to tell that.

File: xml/System.IO/FileSystemInfo.xml
Comment (sabarish98) [reply xml/System.IO/FileSystemInfo.xml]: Added the value

File: xml/System.IO/FileSystemInfo.xml
Comment (sabarish98) [reply xml/System.IO/FileSystemInfo.xml]: Incorporated the changes as suggested : )

**Review by gewarren — APPROVED** [review]
Thanks, LGTM!

**Comment by sabarish98** [comment]
@gewarren : What more do we need to do to get this change merged. Do we need another review?

**Comment by gewarren** [comment]
> @gewarren : What more do we need to do to get this change merged. Do we need another review?

I'd like to get a review form @Jozkee before we merge.

File: xml/System.IO/FileSystemInfo.xml
Comment (Jozkee) [inline xml/System.IO/FileSystemInfo.xml:1321]:
```suggestion
        <summary>Gets the target path of the link located in <see cref="P:System.IO.FileSystemInfo.FullName" />, or <see langword="null" /> if this <see cref="T:System.IO.FileSystemInfo" /> instance doesn't represent a link.</summary>
```
Do not specify symbolic link as this API also works with junctions which are another kind of link only supported by NTFS.

File: xml/System.IO/FileSystemInfo.xml
Comment (Jozkee) [inline xml/System.IO/FileSystemInfo.xml:1322]:
```suggestion
        <value>The target path of the link located in <see cref="P:System.IO.FileSystemInfo.FullName" />, or <see langword="null" /> if this <see cref="T:System.IO.FileSystemInfo" /> instance doesn't represent a link.</value>
```

File: xml/System.IO/FileSystemInfo.xml
Comment (sabarish98) [reply xml/System.IO/FileSystemInfo.xml]: Noted @Jozkee. Made the change

File: xml/System.IO/FileSystemInfo.xml
Comment (sabarish98) [reply xml/System.IO/FileSystemInfo.xml]: Changed this as well

**Pull request merged**
